map/routing_manager.cpp

```diff
@@ -1089,6 +1089,27 @@ void RoutingManager::SetUserCurrentPosition(m2::PointD const & position)
   }
 }
 
+std::string GetNameFromPoint(RouteMarkData rmd)
```

Comment:
No need to copy rmd here. static or anonymous namespace {} will not expose the function to the outside world.
```suggestion
static std::string GetNameFromPoint(RouteMarkData const & rmd)
```

Reply:
Fixed